tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ng.html

```diff
@@ -18,10 +18,8 @@
 <tb-data-table
   [headers]="columnHeaders"
   [sortingInfo]="sortingInfo"
-  [columnCustomizationEnabled]="columnCustomizationEnabled"
   (sortDataBy)="sortDataBy.emit($event)"
-  (orderColumns)="orderColumns($event)"
-  (removeColumn)="removeColumn.emit($event)"
```

Comment:
scalar tables shouldn't be able to remove columns, just hide them. But there will be no functional change atm because this is currently disabled.

Reply:
I agree that showing both "hide" and "remove" buttons is also reasonable, but showing both in the same context menu might make a lot of users wondering what the difference is. Meanwhile, allowing only one of these functionalities per table _might_ make the distinction a bit clearer.

In any case, I think it's important to gather more feedback from the team and from users on what the ideal context menu setup should be. It should be trivial to add/remove items later as necessary.

Reply:
We definitely shouldn't show both in the same context menu. I'm not sure splitting the functionality by table will be clearer for users.